### ISSUE TITLE
Q: How to properly mount fileshare with local(docker-desktop) kubernetes

### ISSUE THREAD

**Issue by AvinashRamireddy31** [andyzhangx/demo#9]
Hi,

My projects involves azure fileshare integration. 
It is working fine in AKS cluster. I would like to replicate the same in local environment for developers using docker-desktop with kubernetes enabled but fileshare is currently is not working because it is provisioned from azure. 

I tried [https://github.com/andyzhangx/demo/tree/master/linux/azurefile](https://github.com/andyzhangx/demo/tree/master/linux/azurefile) with static provisioning because I already have fileshare.
1.Created azure-secret with storage name and key. 
2.Then edited nginx-pod-azurefile-static.yaml file with my sharename. This is throwing ERROR "**.......bad option; for several filesystems (e.g. nfs, cifs) you might need a /sbin/mount.<type> helper program**." 

**How would I link my local setup with azure fileshare for my pod deployment?** I am feeling that I have to change something in mount options.

FYI: I am running docker-desktop on windows machine with WSL2 enabled

**Comment by andyzhangx** [comment]
I think first you should make sure `mount -t cifs ...` works on your local env, it requires `cifs-utils` package

**Comment by AvinashRamireddy31** [comment]
Yes, I am trying to mount using cifs-utils. Found really [interesting article](https://chemejon.wordpress.com/2021/07/21/how-to-mount-cifs-windows-share-to-kubernetes-pod-with-flexvolume/) on that.

But problem is I need to install cifs-utils inside docker-dekstop kubernetes node (shell).
I tried to node shell using Lens  and getting below error.

![image](https://user-images.githubusercontent.com/44237026/152113776-5dd2ace8-554a-480f-b873-58a4ec4d64b4.png)

Looks like it's root/sudo previleges issue. I am not sure what command I should use to node exec with privileges. Lens didn't provide how it's _exec_ing into the node.

**So how would I exec into the node and install cifs-utils in docker-desktop kubernetes?**

**Comment by andyzhangx** [comment]
I am not familiar with docker-desktop kubernetes
